src/content/en/fundamentals/accessibility/02-semantics-bultin/sembuiltin-three.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: shared/narrow
+title: "Affordances"
+description: "Introduction to affordances"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 3
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+When we use a man-made tool or device, we typically look to its form and design to give us an idea of what it does and how it works. An *affordance* is any object that offers, or affords, its user the opportunity to perform an action. The better the affordance is designed, the more obvious or intuitive its use.
+
+Some affordances can be misleading. A common example is a door with a U-shaped handle that most users expect to pull, but that only responds to a push. Other affordances, such as a door with a flat plate that is clearly meant to be pushed, are more direct and obvious.
```

Comment:
I might suggest replacing this paragraph with the teapot analogy Alice used in the course https://classroom.udacity.com/courses/ud891/lessons/7962141425/concepts/79612617560923#

If you can turn up a teapot image we could include that would be rad.